contents/javascript/Javascript-framework-Angular.md

```diff
@@ -0,0 +1,187 @@
+<frontmatter>
+  title: Introduction to Angular
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Angular
+
+**Author: [Ronak Lakhotia](https://github.com/RonakLakhotia)**                                                              
+Reviewers: [Tan Heng Yeow](https://github.com/tanhengyeow), [Jacob Li PengCheng](https://github.com/jacoblipech)
+
+## What Is Angular?
+
+Angular is one of the popular Javascript frameworks used for building web applications.
+The framework aims to make development and testing of enterprise web applications easier. Angular is maintained by Google, as well as a community of developers.
+
+>Angular started as a side project. Back in 2009, Miško Hevery and Adam Abrons released a project that would help developers build web applications using simple HTML tags. The name “Angular” comes from the angle brackets, or < >, that surround all HTML tags.
+>
+<sub>Miško described the idea behind the framework in an <a href="https://www.infoworld.com/article/2612801/javascript/whats-so-special-about-googles-angularjs.html">interview done in 2013</a></sub>
+
+## Why learn Angular?
```

Comment:
This heading (i.e., _Why_ ...) implies you will be listing advantages of Angular. Yet the contents feel like you are describing the characteristics, which should be part of WHAT.

Divide the article into clear sections.
* What is Angular -- what it does, what it looks like, its main characteristics
* Why Angular -- list advantages (and disadvantages), how it compares with others
* How to get started -- explain a learning path

Reply:
Thanks for the pointer. 👍